Makefile

```diff
@@ -44,6 +44,9 @@ deploy: check-env manifests
 	cd config/controller && kustomize edit set image controller=$(IMAGE)
 	kustomize build config/default | kubectl apply -f -
 
+helm-deploy: check-env manifests
+	helm upgrade -i appmesh-controller config/helm/appmesh-controller --namespace appmesh-system --set image.repository=$(REPO) --set image.tag=$(VERSION)
```

Comment:
How about allowing to set `preview` flag as well?

Reply:
+1, good idea. Let me expose it